src/server/utils/isCancellable.js

```diff
@@ -0,0 +1,22 @@
+import { recentPayment } from './recentPayment';
+
+/**
+ * Return false if any of the conditions fail, meaning the payment
+ * should be unable to be cancelled
+ * @param {string} paymentStatus whether the penalty is PAID or UNPAID
+ * @param {boolean} activePenalty whether the penalty is enabled in the database
+ * @param {float} paymentStartTime when (if) the user began paying the penalty
```

Comment:
Change to type Number and I think explain it's a Unix Timestamp of the payment start date. Should it be `undefined` or `null` if it doesn't exist?